### PR TITLE
Vinson pipeline branch

### PR DESCRIPTION
**Pull request by DSMYTH0**

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+name: de-project-vinson
+
+on:
+  push:
+    branches:
+      - '**'
```
Comment (AndreaBirouk) [inline .github/workflows/deploy.yml:6]: is this going to be main later and you just do it this way now for testing purposes?

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        uses: actions/checkout@v4
+      - name: Set Up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+      - name: Install pip
+        uses: BSFishy/pip-action@v1
+        with:
+          requirements: requirements.txt
+      - name: Run Create Env
```
Comment (AndreaBirouk) [inline .github/workflows/deploy.yml]: this needs to be make create-environment and then add a new point  Install requirements and make requirements

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (DSMYTH0) [reply .github/workflows/deploy.yml:6]: According to Elliot - it should make the CI/CD deploy on every push, not just pushes to the main branch

File: .github/workflows/deploy.yml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        run: make requirements
+      - name: Run Dev Setup
+        run: make dev-setup
+      - name: Run All Checks
+        run: make run-checks
+
+
+  deploy-terraform:
+    name: deploy-terraform
```
Comment (AndreaBirouk) [inline .github/workflows/deploy.yml:28]: this is deploying extract lambda

**Review by AndreaBirouk — APPROVED** [review]

**Pull request merged**